package.json

```diff
@@ -100,5 +97,25 @@
     "ts-loader": "^9.5.1",
     "typescript": "^5.0.4",
     "webpack": "^5.90.3"
+  },
+  "packageManager": "pnpm@8.15.6",
+  "engines": {
+    "node": "^20.0.0",
+    "pnpm": "^8.15.6"
+  },
+  "volta": {
+    "node": "20.12.2",
+    "pnpm": "8.15.6"
+  },
+  "pnpm": {
+    "overrides": {
+      "@types/react": "17.0.2",
+      "@types/react-dom": "17.0.2"
+    },
+    "allowedDeprecatedVersions": {
+      "@babel/plugin-proposal-object-rest-spread": "7.12.1",
+      "core-js": "2.6.12",
+      "trim": "0.0.1"
+    }
```

Comment:
Added these to silence deprecation warnings when running `pnpm install`, but these should be fixed by upgrading respective dependencies using these.